tests/integration/cases/micronaut-projects_micronaut-test/report_schema.json

```diff
@@ -0,0 +1,110 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "macaron-json-report-schema",
+    "title": "Report",
+    "type": "object",
+    "properties": {
+        "metadata": {
+            "type": "object"
```

Comment:
Same comment regarding the nested properties.

Reply:
I purposely left out the properties of this `metadata` and the `target.info` fields because we don't have any guarantee that its content is stable and these fields are somewhat ignored in the old JSON compare script. For example, the content of `target.info` can have null for the repository path because the PURL provided doesn't have any repository. 
I can definitely add the schema for those fields as we only use this schema specifically for this test case, where we have a deterministic set of parameters and expectation.

Reply:
> the content of `target.info` can have null for the repository path

Even if the `repository path` is null, the JSON output should still have this key, and regardless of whether we have been checking these properties in the original comparison script or not, we should check them now because users might be consuming them directly from the JSON outputs.

Reply:
I have improved the schema in this commit - https://github.com/oracle/macaron/pull/824/commits/b80bc763e1b4b58a6f2ada5d5921a6475e4b4ccd